source/expressions.tex

```diff
@@ -5954,17 +5954,23 @@
 
 \pnum
 If the composite pointer type is an object pointer type,
```

Comment:
Hm. We now only "bring them to their composite pointer type" (and thereby introduce the "composite pointer type") when one of the operands is of object pointer type. And in that case, the composite pointer type is *always* an object pointer type. So what we really mean here now is "If we did that stuff in the previous paragraph". So maybe:

```suggestion
In this case,
```

Reply:
Agreed. Fixed.